flang/test/Lower/OpenMP/parallel-reduction-private.f90

```diff
@@ -0,0 +1,30 @@
+! Test that reductions and delayed privatization work properly togehter. Since
+! both types of clauses add block arguments to the OpenMP region, we make sure
+! that the block arguments are added in the proper order (reductions first and
+! then delayed privatization.
+
+! RUN: bbc -emit-hlfir -fopenmp --openmp-enable-delayed-privatization -o - %s 2>&1 | FileCheck %s
```

Comment:
Is this patch accessible through `bbc` at the moment? Not through `flang-new`? I am just curious because we have been not been testing with `bbc` (without testing for `flang-new`) for a while now. Is it something to do with `--openmp-enable-delayed-privatization`?